### ISSUE TITLE
Reduced object sharing

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7319]
**Preferred term label:**
Reduced object sharing

**Synonyms**

**Definition (free text, please give PubMed ID)**
An absence or reduction in object sharing. Object sharing is an early emerging social skill in which a child will share objects with parents or peers during object play. Object sharing typically develops within the first two years of life.
PMID: 26803417

**Parent term (use hpo.jax.org/app)**
Abnormal social behavior
HP:0012433

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
Added as HP:4000085
